File: sentimental-mario-less/mario.py
# ---------------------------------
# Markus Dwiyanto Tobi Sogen
# CS50 for teachers
# Indonesia
# mario-less.py
# ---------------------------------
# TODO
from cs50 import get_int
while True: #cek if user input number true or false
    height = get_int("Height:  ")
    # try check number input from user
    if height > 0 and height < 9:
        break
# row
for row in range(height):
    for space in range(height - row -1, 0, -1): # try make space
        print(" ", end="")
    for hash in range(row + 1): # try make hash
        print("#", end ="")
    print("\n", end ="")

File: Test-csxx50/test/js/mario.js
//---------------------------------
//Markus Dwiyanto Tobi Sogen
//CS50 for teachers
//Indonesia
//mario-more.js
//---------------------------------

const readlineSync = require('readline-sync');

const height = Number(readlineSync.question('Height: '));

while (height < 1 || height > 8) {
  height = Number(readlineSync.question('Height: '));
}

for (let row = 0; row < height; row++) {
    for (let column = row + 1; column < height; column++) {
      process.stdout.write(' ');
    }
    for (let hashes = height + row + 1; hashes > height; hashes--) { // right
      process.stdout.write('#');
    }
    process.stdout.write('  ');
    for (let hashes = height + row + 1; hashes > height; hashes--) { // left
      process.stdout.write('#');
    }
    console.log(); // add newline at the end of each row
  }

  /*I use calls to process.stdout.write without a newline character can result in
  a single long line of output. This is why the output may not look like a pyramid.
  and adding console.log() at the end of each row, we ensure that each row is printed
  on a new line, and the output will be a pyramid
  */